src/DynamoCore/Models/WorkspaceModel.cs

```diff
@@ -867,7 +867,9 @@ internal void UpdateModelValue(IEnumerable<Guid> modelGuids, string propertyName
 
             var retrievedModels = GetModelsInternal(modelGuids);
             if (!retrievedModels.Any())
-                throw new InvalidOperationException("UpdateModelValue: Model not found");
+            {
+                return;
+            }
```

Comment:
The fix should be in `CodeBlockEditor` level, it can't be here. This exception is meant to catch problematic callers like the one in this defect.